INSTALL.md

```diff
@@ -226,6 +226,44 @@ python setup.py build --abs-rpath
 
 **Note:** when building `scikit-learn-intelex` from source with this option, it will use the oneDAL library with which it was compiled. oneDAL has dependencies on other libraries such as TBB, which is also distributed as a python package through `pip` and as a `conda` package. By default, a conda environment will first try to load TBB from its own packages if it is installed in the environment, which might cause issues if oneDAL was compiled with a system TBB instead of a conda one. In such cases, it is advised to either uninstall TBB from pip/conda (it will be loaded from the oneDAL library which links to it), or modify the order of search paths in environment variables like `${LD_LIBRARY_PATH}`.
 
+### Building with ASAN
+
+In order to use the address sanitizer (ASAN) together with `scikit-learn-intelex`, it's necessary to:
```

Comment:
```suggestion
In order to use AddressSanitizer (ASAN) together with `scikit-learn-intelex`, it's necessary to:
```
Based on how they refer to it in https://github.com/google/sanitizers/wiki/addresssanitizer